lib/lightning_web/channels/attempt_channel.ex

```diff
@@ -182,6 +187,12 @@ defmodule LightningWeb.AttemptChannel do
     end
   end
 
+  def handle_in("attempt:log", %{"message" => message}, socket)
+      when message == nil or message == [nil] do
+    {:reply, {:error, %{errors: %{message: ["This field can't be blank."]}}},
+     socket}
+  end
+
```

Comment:
I think we should be handling this in a changeset (and the ChangesetJSON module) like the rest of the `handle_in`s.

I'm curious about why this is needed - was it done before ff5f6f325231254511ba88a64a322d13fd27e247? That commit fixed coercing `nil` and `[nil]`. While it doesn't reject the message, it safely converts it to `null` - I don't think we want to stand in the way of controlling what is and isn't a valid message from the worker - we should either drop it or convert it. It's entirely possible a user tries to `console.log(a)` and a is `null`.

Reply:
I have got an error with message = [nil] on the payload, moved the validation to the changeset.

Reply:
Ah ok, since you rebased the other day this is no longer required (i.e. in the changeset validation as well).

The `LogMessage` Ecto type handles coercing from `nil|[nil]` into `"null"`.